src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,37 @@
 
 #include "t07_ip.h"
 #include <iostream>
-
+#include <string>
 
 using namespace std;
 
-int t07_ip() {
+int t07_ip() 
+{
+    string s;
+    getline (cin, s);
+    int b,d=0;
+    string c;
+    b = 0;
+    for (int i = 0; i < s.length(); i++) {
+        int a=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'a' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4607778)